Microsoft.Toolkit.Wpf.UI.XamlHost/WindowsXamlHostBase.Visibilty.cs

```diff
@@ -0,0 +1,10 @@
+// Licensed to the .NET Foundation under one or more agreements.
+// The .NET Foundation licenses this file to you under the MIT license.
+// See the LICENSE file in the project root for more information.
+
+namespace Microsoft.Toolkit.Wpf.UI.XamlHost
```

Comment:
nit: Comment as to why these .visible.cs files are required by internal test infrastructure.